ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetTape.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * *****************************************************************************
+ *   Copyright 2014-2016 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ***************************************************************************
+ */
+
+package com.spectralogic.ds3cli.command;
+
+import com.google.common.collect.ImmutableList;
+import com.spectralogic.ds3cli.Arguments;
+import com.spectralogic.ds3cli.View;
+import com.spectralogic.ds3cli.ViewType;
+import com.spectralogic.ds3cli.models.TapeResult;
+import com.spectralogic.ds3cli.views.cli.GetTapeView;
+import com.spectralogic.ds3cli.views.json.DataView;
+import com.spectralogic.ds3client.commands.spectrads3.GetTapeSpectraS3Request;
+import org.apache.commons.cli.Option;
+
+import static com.spectralogic.ds3cli.ArgumentFactory.ID;
+
+public class GetTape extends CliCommand<TapeResult> {
+
```

Comment:
Only question here is replacing this command with accepting an optional ID param on GetTapes. Where the plural and singular commands both return the same object (and a collection of it), I have tried to keep it in one command and have result construct a collection of one.